packages/next-plugin-sentry/package.json

```diff
@@ -7,16 +7,15 @@
   },
   "nextjs": {
     "name": "Sentry",
-    "required-env": [
-      "SENTRY_DSN",
-      "SENTRY_RELEASE"
-    ]
+    "required-env": []
```

Comment:
Why are there no longer any required envs?

Reply:
See the PR description.

Reply:
We should fix the bug with the key please instead of bypassing it!

Reply:
I totally agree. This PR is the effect of PoC work, and it was necessary to go around this limitation for demo purposes.